core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -672,9 +672,13 @@ enum class UniqueType(
     ConditionalOurUnitOnUnit("when [mapUnitFilter]", UniqueTarget.Conditional), // Same but for the unit itself
     ConditionalUnitWithPromotion("for units with [promotion]", UniqueTarget.Conditional),
     ConditionalUnitWithoutPromotion("for units without [promotion]", UniqueTarget.Conditional),
-    ConditionalVsCity("vs cities", UniqueTarget.Conditional),
+
+    ConditionalVsCity("vs [cityFilter] cities", UniqueTarget.Conditional),
+    @Deprecated("as of 4.10.9", ReplaceWith("vs [all] cities"))
+    ConditionalVsCityOld("vs cities", UniqueTarget.Conditional),
```

Comment:
Just realized, you forgot to leave behind a temporary conditional logic for the older legacy version